test/test_helper.rb

```diff
@@ -56,6 +56,8 @@ class ActiveSupport::TestCase
   include UrlHelpers
   extend GovspeakValidationTestHelper
 
+  parallelize(workers: 4)
```

Comment:
Worth saying I plucked this number out of thin air. It would be worth experimenting with a few different values between say 2 and 10 to see what gives the best time and consistency.

Reply:
You can do `parallelize workers: :number_of_processors`  https://github.com/alphagov/smart-answers/blob/151d10c3fe912902e57bdfe4722463f642085f1d/test/test_helper.rb#L31

Reply:
@kevindew Ooh good to know! I'll add this to the work in https://github.com/alphagov/whitehall/pull/6169 after merging this PR :)